src/cryptoadvance/specter/templates/device/device.jinja

```diff
@@ -1,6 +1,11 @@
 {% extends "base.jinja" %}
 {% block main %}
-	<h1>Public keys of {{ device.name }}</h1>
+	<h1>{{ device.name }}</h1>
+	<form action="./" method="POST">
+		{% include "device/components/device_type.jinja" %}
+		<button type="submit" class="btn hidden" id="settype" name="action" value="settype" style="float: right;">Update</button>
+	</form>
```

Comment:
"Update" button is not aligned with the rest of the elements.

What do you think about doing it like this?
```html
<form action="./" method="POST">
	<div class="row">
		{% include "device/components/device_type.jinja" %}
		<button type="submit" class="btn hidden" id="settype" name="action" value="settype" style="height:15px; align-self:center;">Update</button>
	</div>
</form>
```

Reply:
Oh I didn't notice that as I use Safari, on which it looks the same as with the fix, I should remember to test on Chrome too...